example-config.yaml

```diff
@@ -1,7 +1,15 @@
+# You will need to use a custom emoji here for when the bot is processing the image.
+# Get rid of the "a" if you are not using an animated emoji
```

Comment:
Using unicode emoji is also possible.

Reply:
I just realized that. I'll edit the comment then.